### DISCUSSION TITLE
Unable to access github inside the linux VM (default configurations)

### DISCUSSION THREAD

**Discussion by karthickradhakrishnan** [utmapp/UTM#6366] [UTM Help (macOS)]
Unable to access github inside the Linux VM which is running on top of M1 Mac host. Tried with terminal and console and still facing the same issue. Tried updating the settings but no luck

<img width="1658" alt="Screenshot 2024-05-16 at 6 44 54 AM" src="https://github.com/utmapp/UTM/assets/107867705/d4554fc9-56dc-4f9a-92a8-d4b8c57d51fb">

**Comment by osy** [comment]
Does any other website work?

**Reply by karthickradhakrishnan** [reply]
Tried accessing multiple websites and notice that few of them are not working.

Working: Google, youtube, facebook 
Not working: Github, reddit

**Reply by osy** [reply]
Are you sure it's not a Linux problem? This feels like a Linux problem to me

**Reply by karthickradhakrishnan** [reply]
Exploring the network but couldn't crack anything so tried checking in here

**Reply by umanwizard** [reply]
GItHub and Reddit are IPv4-only. The others support IPv6. Perhaps something is wrong with your IPv4 stack.

**Reply by drew138** [reply]
happening to me as well on a macos vm

**Reply by only-a-user** [reply]
I am experiencing the same issue, when running a macOS vm. Specifically github and reddit also do not work.
